addon-api/popup/Popup.js

```diff
@@ -0,0 +1,17 @@
+export default class Popup {
+  /**
+   * Whether the popup is displayed fullscreen.
+   * @type {boolean}
+   */
+  get isFullscreen() {
+    return window.parent === window;
+  }
+
+  /**
+   * Whether the user has enabled light mode on Scratch Addons settings.
+   * @type {boolean}
+   */
+  get isLightMode() {
+    return scratchAddons.isLightMode;
+  }
+}
```

Comment:
Maybe make this just `fullscreen` and `lightmode`?

Reply:
We have `isLoggedIn`. We can keep this